files/vmq-admin

```diff
@@ -21,4 +21,4 @@ cd $RUNNER_BASE_DIR
 # Identify the script name
 SCRIPT=`basename $0`
 
-$NODETOOL rpc vmq_server_cli command $SCRIPT $@
+$NODETOOL rpc_infinity vmq_server_cli command $SCRIPT $@
```

Comment:
we should revert it to rpc. As the rpc_infinity I planned to use for the repl part.